test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -374,7 +374,20 @@ export $envVarName='$guid'
     }
 
     Context "Pipe to/from powershell" {
-        $p = [PSCustomObject]@{X=10;Y=20}
+        BeforeAll {
+            if ($null -ne $PSStyle) {
+                $outputRendering = $PSStyle.OutputRendering
```

Comment:
I'm not quite sure what this dance is doing, but I should think that if we are going to call a native executable we should automatically set up plain text rendering.

Reply:
If `PSAnsiRendering` experimental feature isn't enabled `$PSStyle` doesn't exist.  So this is handling that case if this test is being run locally and putting back the user's custom outputrendering.  Think of it as no different than putting back `$ErrorActionPreference` when run locally.